src/Ztm.WebApi/HttpResponseExceptionFilter.cs

```diff
@@ -0,0 +1,36 @@
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.Filters;
+using Newtonsoft.Json;
+
+namespace Ztm.WebApi
+{
+    class ErrorDetails
```

Comment:
If we follow my idea seem like we don't need this.

Reply:
removed